### PR TITLE
Method references supported in transform and fromExpression

### PR DESCRIPTION
**Pull request by AbelPelser**

### CODE REVIEW

**Comment by stefankoppier** [comment]
Sonar is a false positive (secrets cannot be used in forked PR's...) I'm gonna disable SonarQube

**Comment by stefankoppier** [comment]
Can you also add this change to the changelog in `website/src/changelog.md` with the date set to `tbd`

**Comment by AbelPelser** [comment]
> Can you also add this change to the changelog in `website/src/changelog.md` with the date set to `tbd` and version `0.10.0`?

Yes, done

**Comment by stefankoppier** [comment]
> > Can you also add this change to the changelog in `website/src/changelog.md` with the date set to `tbd` and version `0.10.0`?
> 
> Yes, done

Sorry, I just did the same 😓 I also removed Sonar in branches, so the next build after a merge with main should succeed.

**Review by stefankoppier — CHANGES_REQUESTED** [review]

File: compiler-plugin/src/test/kotlin/tech/mappie/testing/objects/MethodReferenceTest.kt [hunk 1]
```diff
@@ -0,0 +1,75 @@
+package tech.mappie.testing.objects
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.io.TempDir
+import tech.mappie.testing.compilation.compile
+import tech.mappie.testing.loadObjectMappieClass
+import java.io.File
+
+class MethodReferenceTest {
```
[the remark this answers is not in the document]
Comment (AbelPelser) [reply compiler-plugin/src/test/kotlin/tech/mappie/testing/objects/MethodReferenceTest.kt]: Done, same goes for the other comments now I think

**Comment by stefankoppier** [comment]
LGTM. Thanks for the PR!

**Pull request merged**